tested/main.py

```diff
@@ -30,10 +33,19 @@ def run(config: DodonaConfig, judge_output: IO):
     _, ext = os.path.splitext(config.test_suite)
     is_yaml = ext.lower() in (".yaml", ".yml")
     if is_yaml:
+        if language:
+            textual_suite, missing_keys = run_translation_with_str(
```

Comment:
This method name is a bit odd. 'running' translations isn't good English

Maybe `apply_translations_in_yaml_string` or maybe just `apply_translations`